### PR TITLE
Validation for CalculatorInternalController

### PR DESCRIPTION
**Pull request by udaydefra**

### CODE REVIEW

**Review by tim-powell-eviden — DISMISSED** [review]

File: src/EPR.Calculator.API/Program.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@
 builder.Services.AddScoped<ICalcResultsExporter<CalcResult>, CalcResultsExporter>();
 builder.Services.AddScoped<ICalcResultLapcapDataBuilder, CalcResultLapcapDataBuilder>();
 builder.Services.AddScoped<ICalcResultSummaryBuilder, CalcResultSummaryBuilder>();
-builder.Services.AddScoped<IStorageService, BlobStorageService>();
+builder.Services.AddScoped<IStorageService, LocalFileStorageService>();
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API/Program.cs:45]: Is this change required?

**Review by johnbenjamin-defra — APPROVED** [review]

**Review by tim-powell-eviden — APPROVED** [review]
